### ISSUE TITLE
Add wind chill value.

### ISSUE THREAD

**Issue by richierubin** [martykan/forecastie#576]
It's -3F today and I thought no biggie, but then got blasted with 15mph wind which was brutal.  Any chance to add/calculate wind chill factor?  That's always nice to see, thanks!

[Potential calculation?](https://sciencing.com/calculate-wind-chill-factor-5981683.html)

**Comment by robinpaulson** [comment]
Yes, a valid request, very useful to have. There have been a couple of requests previously, I think the remaining issue is called 'real feel temperature'. I'll close thus as it's a dupe.

**Comment by robinpaulson** [comment]
Huh, I can't find it now. There is some talk about it in #397

**Comment by Essjayess** [comment]
Yes! Would very much love to have this feature.
